README.md

```diff
@@ -1,34 +1,65 @@
 
-First, run the development server:
+  
+This Next.js starter project fully integrates with dynamic sample content from your ButterCMS account, including main menu, pages, blog posts, categories, and tags, all with a beautiful, custom theme with already-implemented search functionality. All of the included sample content is automatically created in your account dashboard when you sign up for a free trial of ButterCMS.
 
+Once created, this project can be easily and quickly deployed to Vercel from the CLI ([see instructions below](#deploy-on-vercel))
+
+
+## Installation
+
+First, install the dependencies by running one of these commands, depending on your current setup:
 ```bash
-npm run dev
+npm install
 # or
-yarn dev
+yarn install
 ```
+### Set API Token
 
-Open [http://localhost:3000](http://localhost:3000) with your browser to see the result.
-
-You can start editing the page by modifying `pages/index.js`. The page auto-updates as you edit the file.
+To fetch your ButterCMS content, add your API token as an environment variable. You can find it in your butterCMS account under settings -> API Token. If you don't already have a ButterCMS account, you can get one free [here](https://buttercms.com/join/). Set `NEXT_PUBLIC_BUTTER_CMS_API_KEY=<Your API Token>` in `.env`.
 
-[API routes](https://nextjs.org/docs/api-routes/introduction) can be accessed on [http://localhost:3000/api/hello](http://localhost:3000/api/hello). This endpoint can be edited in `pages/api/hello.js`.
+```bash
+$ echo 'NEXT_PUBLIC_BUTTER_CMS_API_KEY=<Your API Token>' >> .env
+```
 
-The `pages/api` directory is mapped to `/api/*`. Files in this directory are treated as [API routes](https://nextjs.org/docs/api-routes/introduction) instead of React pages.
+### Run the local server
+You'll want to run server any time you want to view the project locally:
 
-## Learn More
+```bash
+npm run dev
+# or
+yarn dev
+```
+After running the command, you should see a message:
+```
+ready - started server on 0.0.0.0:3000, url: http://localhost:3000
+```
+Congratulations! Your starter project is now live and can be viewed at [http://localhost:3000](http://localhost:3000).
 
-To learn more about Next.js, take a look at the following resources:
+To quit the server press CONTROL-C.
+#### Building the app
 
-- [Next.js Documentation](https://nextjs.org/docs) - learn about Next.js features and API.
-- [Learn Next.js](https://nextjs.org/learn) - an interactive Next.js tutorial.
+```bash
+npm run build
+```
 
-You can check out [the Next.js GitHub repository](https://github.com/vercel/next.js/) - your feedback and contributions are welcome!
+or
 
-## Deploy on Vercel
+```bash
+yarn build
+```
 
-The easiest way to deploy your Next.js app is to use the [Vercel Platform](https://vercel.com/new?utm_medium=default-template&filter=next.js&utm_source=create-next-app&utm_campaign=create-next-app-readme) from the creators of Next.js.
```

Comment:
Can I ask why this information was cut out of the readme? I think if we're taking it out, we should replace it with something that explains at least a very basic functionality of Vercel here, e.g.,: "If you want to deploy your app live, you can do that for free in just a few clicks with Vercel."

Reply:
Good point. I've returned the whole paragraph back